HAL Archives Ouvertes.js

```diff
@@ -90,34 +98,33 @@ function doWeb(doc, url) {
 			for (var i in items) {
 				articles.push(i);
 			}
-			Zotero.Utilities.processDocuments(articles, scrape)
+			Zotero.Utilities.processDocuments(articles, scrape);
+			return true;
```

Comment:
We don't actually check the return value of the `selectItems` callback (anymore?), so the `consistent-return` fix here is just to change the first `return true;` to `return;`.